src/main/docker/Dockerfile.jvm.staged

```diff
@@ -57,12 +57,12 @@ RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8081 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 # We make four distinct layers so if there are application changes the library layers can be re-used
-COPY --from=0 --chown=1001 /build/target/quarkus-app-tmp/*.jar /deployments/export-run-artifact.jar
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*lib/ /deployments/lib/
+COPY --from=0 --chown=1001 /build/target/copyhack /build/target/quarkus-app-tmp/*.jar /deployments/export-run-artifact.jar
+COPY --from=0 --chown=1001 /build/target/copyhack /build/target/*quarkus-app/*lib/ /deployments/lib/
 # Overwrite the uber jar if package type is normal
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*.jar /deployments/export-run-artifact.jar 
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*app/ /deployments/app/
-COPY --from=0 --chown=1001 /build/target/*quarkus-app/*quarkus/ /deployments/quarkus/
+COPY --from=0 --chown=1001 /build/target/copyhack /build/target/*quarkus-app/*.jar /deployments/export-run-artifact.jar 
```

Comment:
If there is an uberjar won't this result in the uberjar being overwritten by 'copyhack'? And if there is not an uberjar how would this work with two src resources and a single file as the dest?

Reply:
I just tried this and for uberjar I get:

"exec java -Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8081 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+ExitOnOutOfMemoryError -cp . -jar /deployments/export-run-artifact.jar
Error: Invalid or corrupt jarfile /deployments/export-run-artifact.jar"

Reply:
Yeah, totally bogus.
I think for the run jar, we should put the uber-jar in the same place as the fast jar and only do a single copy.
For the rest the file hack works. There may be other tricks to experiment with.

Reply:
I have tha approach here: https://github.com/devfile-samples/devfile-sample-code-with-quarkus/pull/15